json/testnet-bitcoin-config.json

```diff
@@ -1,13 +1,13 @@
 {
```

Comment:
I feel we should have separate config files used specifically for docker, because these are still useful for directly, non-containerized installs. Maybe just create them in the docker/ folder?

Reply:
Potentially, but unless people are careful remembering there are two places to update the value, this problem occurs.

Testnet config was updated here with the latest release and was not updated here meaning the docker compose script does not run at the moment as reported in #99 and #100.

I'd prefer we dont have two authoritative places for the same value.

Reply:
@tplooker, Agree with you on not wanting to duplicate and maintain the same config in multiple places in principle. But in this case the change to mongo connection string seems to be docker specific which would great existing standalone install guide? Also, do you know how the bitcoin RPC username and password are propagated into this configuration file?